specify/specify7
# Move `without preparations` and `add unassociated item` buttons to the left of Interactions dialog

Fixes #4394



### Checklist

      and self-explanatory (or properly documented)

### Testing instructions


- Go to Interactions
- Click on Loan or Gift
  - [ ] Verify `Without preparations` button is moved to the left of the dialog and is gray
  - Click on `By choosing a record set`
    - Click on any record set
    - If record set has no preparations, click on continue without preparations
    - Click on + icon next to Loan preparations
    - [ ] Verify `Add unassociated item` button is also moved to the top of the dialog and is gray


![image](https://github.com/specify/specify7/assets/16229739/9127dac5-cf15-44aa-928c-0fefa305f099)
![image](https://github.com/specify/specify7/assets/16229739/ce8fee53-df1b-4a4c-87f8-f7b518667178)
